src/jbuild

```diff
@@ -0,0 +1,17 @@
+(library
+ ((name        mirage_net)
+  (public_name mirage-net)
+  (libraries   (bytes result mirage-device))
+  (modules     (Mirage_net))
+  (flags       (:standard -w "A-4-41-44" -warn-error "+1..49" -safe-string))
```

Comment:
In this case I coped the contents of the `_tags` file: `true: warn_error(+1..49), warn(A-4-41-44)`. I'm not very familiar with the semantics of this though. Is it not safe to use a specified list "1,2,3,...,49" since they're already defined and won't be redefined in future versions of the compiler? I thought the main problem was with the equivalent of `-Wall` which includes future yet-to-be-defined errors.

Wrt to `topkg` could you explain a bit more? I see this:
```
$ topkg build --dev
topkg: unknown option `--dev', did you mean `-d' ?
Usage: topkg build [OPTION]... [BUILD_CONF]...
Try `topkg build --help' or `topkg --help' for more information.
```
(I was never a `topkg` expert). Is your suggestion to use `topkg` and `jbuilder` together? Could you outline what the build and release workflow would look like? Thanks!